code/__DEFINES/subsystems.dm

```diff
@@ -7,6 +7,7 @@
 //Don't run if there is an identical unique timer active
 #define TIMER_UNIQUE		0x1
 //For unique timers: Replace the old timer rather then not start this one
+//if the argumentss to addtimer are the same as an existing timer, it doesn't create a new timer, and returns the id of the existing timer
```

Comment:
i knew cyberboss was a lizard all along

Reply:
You added this for the documentation of `TIMER_OVERRIDE`, not `TIMER_UNIQUE`